src/__init__.py

```diff
@@ -1,15 +0,0 @@
-# Copyright 2013 National Renewable Energy Laboratory (NREL)
-# 
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-# 
-#        http://www.apache.org/licenses/LICENSE-2.0
-# 
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-# 
-# ++==++==++==++==++==++==++==++==++==++==
```

Comment:
Why not? It makes things so much easier to include daktoa.py as a module and not as a package. 
I already told you yesterday that I would din fact remove dakota.py completely since it is more or less just a demo on how to use pyDAKOTA. We have to extend the classes defined here anyway and override most of the functionality here. But this is another issue...